topological_exploration/launch/topological_exploration.launch

```diff
@@ -1,15 +1,23 @@
 <launch>
 
-	<arg name="exploration" pkg="topological_exploration" type="topological_exploration">
-		<param name="sweep_type" value="complete"/>
+	<arg name="schedule_directory" default="/localhome/strands/schedules"/>
+	<arg name="grids_directory" default="/localhome/strands/3dmaps"/>
+	<arg name="sweep_type" default="complete"/>
+	<arg name="tag_node" default="Exploration"/>
+	<arg name="tag_ground_node" default="ExplorationGround"/>
+
+	<node name="exploration" pkg="topological_exploration" type="topological_exploration" output="screen">
+		<param name="tag_node" value="$(arg tag_node)"/> 
```

Comment:
can you change the name of the node to something a bit more descriptive? There are more exploration nodes around, and they're also topological based in a sense